bin/build-gallery.js

```diff
@@ -390,13 +404,13 @@ function minifyAndGenPage (file, bundle) {
     })
 
     closureCompiler.run(function (exitCode, stdOut, stdErr) {
+      console.log('minified ', minFile)
+      console.log('stdout: ', stdOut)
+      console.log('stderr: ', stdErr)
```

Comment:
Outputting this data before attempting to save the file, in case there was a full-blown error that prevented the minified file from being generated in the first place